README.md

```diff
@@ -7,18 +7,18 @@ migration, using data from an existing PostgreSQL, MySQL, SQL Server, Oracle or
 The tool ingests schema and data from either a pg_dump/mysqldump file or directly
 from the source database, and supports both schema and data migration. For schema
 migration, HarbourBridge automatically builds a Spanner schema from the schema
-of the source database. This schema can be customized using the HarbourBridge
-schema assistant. For data migration, HarbourBridge creates a new Spanner
-database using the Spanner schema built during schema migration, and populates
-it with data from the source database.
+of the source database and creates a new Spanner database using the Spanner schema built. This schema can be customized using the HarbourBridge schema assistant. For data migration, HarbourBridge populates
+the spanner database with data from the source database.
 
 For more details on schema customization and use of the schema assistant, see
-[web/README](web/README.md). The rest of this README describes the command-line
+[web/README](webv2/README.md). The rest of this README describes the command-line
 capabilities of HarbourBridge.
 
-HarbourBridge is designed to simplify Spanner evaluation and migration, and in
-particular for migrating moderate-size datasets to Spanner
-(up to about 100GB). Certain features of relational databases, especially those that don't
+HarbourBridge is designed to simplify Spanner evaluation and migration.
+It supports two types of migrations - bulk migration and streaming migration.
+In bulk migration harbourbridge performs end to end migration and is suggested for
+migrating moderate-size datasets to Spanner(up to about 100GB). In case of streaming migration harbourbridge orchestrates datastream and dataflow job to perform the migration and is suggested for larger databases.
```

Comment:
use bullets/headings on this section to cover each type of migration with details. Please start with Low Downtime migrations as that will be the main recommendation